src/control_protocol.rs

```diff
@@ -93,3 +109,19 @@ pub struct SendMessageArgs {
     #[serde_as(as = "serde_with::base64::Base64")]
     pub content: Bytes,
 }
+
+#[serde_as]
+#[derive(Serialize, Deserialize)]
+pub struct SendGlobalRpcArgs {
+    pub id: Option<String>,
+    pub source_dock: Option<Dock>,
+    #[serde_as(as = "serde_with::DisplayFromStr")]
+    pub destination: Fingerprint,
+    pub request: JrpcRequest,
```

Comment:
This works, but I feel like manually formatting a JSON-RPC request is somewhat annoying. I would rather pass in `method` and `args`, and call `call` rather than `call_raw`. Then in the control protocol client CLI, have a syntax like

```
$ earendil control send-global-rpc <fingerprint> <port> ping 12345
12345
```

where we specify the verb "ping", then list the arguments one after the other on the command line, each individually in JSON format but not in one JSON object.

I am not sure how easy it is to make something "variadic" like this in clap though. If it's too hard the present approach is probably fine.